backend/config/settings/base.py

```diff
@@ -120,3 +121,16 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+
+# List of sites from which you can make requests to this project.
+# To enable CORS_ALLOWED_ORIGINS, change CORS_ALLOW_ALL_ORIGINS to False
+CORS_ALLOWED_ORIGINS = [
+    'http://127.0.0.1:8000',
+
+]
+
+CORS_ALLOW_ALL_ORIGINS = True 
+
+CORS_ALLOW_METHODS = list(default_methods) + []
+CORS_ALLOW_HEADERS = list(default_headers) + []
```

Comment:
Please add an empty string, PEP8 =)

Reply:
Empty strings are used to separate logical parts of the code.

Reply:
That was concerning a newline at the end of the file, which is missing.